### PR TITLE
styles: Adds customization example

### LINKED ISSUE #234: v3.4: add theming customization example
When the theming customization will be ready in `invenio-assets/theme`, then:
1. add an example of a LESS file with a CSS customization
2. revert the commented out assets include in the `page.html`:
`{#{{ webpack['{%- endraw -%}{{cookiecutter.project_shortname}}{%- raw -%}-theme.css'] }}#}`

### LINKED ISSUE #265: search: record item link does not work
## Describe the bug

The search page result item does not redirect to the record when clicked. In the HTML there is **no href**.

## Steps to Reproduce
1. Bootstrap from master
2. Create a record
3. Search and click on it

## Expected behavior

Redirect to the landing page

## Screenshots (if applicable)

Screenshot 2020-12-09 at 14 21 37

## Additional context

**SOLUTION**: This is because the `ResultList.Item` component is not being override, and the default `invenio-search-ui` does not contain a link.

### PR DESCRIPTION
**Pull request by jrcastro2**
* closes #234 
* closes #265

### CODE REVIEW

**Comment by lnielsen** [comment]
The idea with the ticket is to provide an example in the docs on how to change some simple things (so not changing the actual code). We could leave the code commented out, so that it's easy to enable.

The example is good, just shouldn't be committed as the default :-)

Also, we should provide an example of customizing the result item of the search app.

File: {{cookiecutter.project_shortname}}/{{cookiecutter.package_name}}/records/templates/semantic-ui/records/search.html [hunk 1]
```diff
@@ -3,8 +3,6 @@
 {%- extends "semantic-ui/invenio_search_ui/search.html" %}
 
 {% block javascript %}
-  <!-- Because loading order of overridables -->
   {{ webpack['{%- endraw -%}{{cookiecutter.project_shortname}}{%- raw -%}-search-app.js'] }}
-  {{ super() }}
```
Comment (lnielsen) [inline {{cookiecutter.project_shortname}}/{{cookiecutter.package_name}}/records/templates/semantic-ui/records/search.html:8]: Question:  Are you sure about this one?

**Review by lnielsen — COMMENTED** [review]
Looks good....just one question about the ``super()`` in the template. What happens if it stays?

File: {{cookiecutter.project_shortname}}/{{cookiecutter.package_name}}/records/templates/semantic-ui/records/search.html (hunk 1 above)
Comment (jrcastro2) [reply {{cookiecutter.project_shortname}}/{{cookiecutter.package_name}}/records/templates/semantic-ui/records/search.html:8]: If it stays, the searchbar is not being displayed. 
image

**Pull request merged**